sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md

```diff
@@ -5,7 +5,7 @@
e Not Found) exception is thrown. </br><ul><li>The Cosmos DB SDK does retry the read request against a second replica for the partition (in the same region) with the specified session token. </li><li>If the second replica also throws a 404: </br><ul><li>If there are additional regions for the Cosmos DB account: </br><ul><li>For a single-write region account, the client retries the request against the write region for the account (if the first request targeted the read region). The client will follow the same path of targeting 1 replica followed by a retry against another replica if the read against the first replica resulted in a 404. </li><li>If all regions have 2 replicas that returned a 404, the exception is bubbled back up to the calling code. </li></ul></li><ul></li></ul></li></ul></li><li>When using Bounded Staleness and Strong Consistency: </br><ul><li>The SDK sends the read request to 2 replicas for the partition in the specified region. </li><li>If one of the 2 replicas returns a 404, data from the second replica is returned by the SDK. </li><li>If both replicas return a 404, the exception is bubbled up to the application. </li><li>The Cosmos DB SDK does NOT retry the read against a remote region is both replicas in the specified region return a 404 for the point read operation. </li></ul></li></ul></li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
5secons might be interpreted as contract. Is that guaranteed?

Reply:
maybe I can change into: will first retry on the current region, then other regions?

Reply:
removed the 5s reference. Changed to: will first retry on the current region few times, then other regions.